arcgis-ios-sdk-samples/Content Display Logic/Storyboard/Main.storyboard

```diff
@@ -203,121 +203,47 @@
-                                                <constraint firstItem="8Vw-U7-C2m" firstAttribute="top" secondItem="QnF-3P-vPe" secondAttribute="bottom" id="tgm-op-afz"/>
-                                                <constraint firstItem="QnF-3P-vPe" firstAttribute="top" secondItem="YQX-QH-TKY" secondAttribute="top" constant="10" id="xQM-70-Ebk"/>
-                                            </constraints>
-                                        </view>
-                                    </subviews>
-                                    <constraints>
-                                        <constraint firstAttribute="trailing" secondItem="YQX-QH-TKY" secondAttribute="trailing" id="0P9-hs-eUl"/>
-                                        <constraint firstItem="YQX-QH-TKY" firstAttribute="leading" secondItem="yKf-rX-5VP" secondAttribute="leading" id="QO5-kC-AfF"/>
-                                        <constraint firstItem="YQX-QH-TKY" firstAttribute="top" secondItem="yKf-rX-5VP" secondAttribute="top" id="gFY-uF-kQa"/>
-                                        <constraint firstAttribute="bottom" secondItem="YQX-QH-TKY" secondAttribute="bottom" id="tUl-Wi-dMJ"/>
-                                    </constraints>
-                                </tableViewCellContentView>
-                            </tableViewCell>
-                            <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" selectionStyle="default" indentationWidth="10" reuseIdentifier="ContentTableCell" rowHeight="80" id="UHI-3v-SDe" customClass="ContentTableCell" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target">
-                                <rect key="frame" x="0.0" y="66" width="375" height="80"/>
-                                <autoresizingMask key="autoresizingMask"/>
                                 <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" tableViewCell="UHI-3v-SDe" id="hjB-p9-kF0">
-                                    <rect key="frame" x="0.0" y="0.0" width="375" height="80"/>
+                                    <rect key="frame" x="0.0" y="0.0" width="307" height="43.666666666666664"/>
                                     <autoresizingMask key="autoresizingMask"/>
                                     <subviews>
-                                        <view contentMode="scaleToFill" placeholderIntrinsicWidth="580" placeholderIntrinsicHeight="62" translatesAutoresizingMaskIntoConstraints="NO" id="jE0-B2-BRg">
-                                            <rect key="frame" x="18" y="12" width="339" height="56"/>
+                                        <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="6" translatesAutoresizingMaskIntoConstraints="NO" id="pVi-Nv-PRP">
```

Comment:
One more thing I noticed. The alignment should be Leading (it shows up as `top` in the storyboard). The way it is now, the labels are sized to fit the entire width of the stack view:

![screen shot 2018-11-30 at 10 58 56 am](https://user-images.githubusercontent.com/2257493/49320536-f6677280-f4b6-11e8-9306-d97ed64a0451.png)

By using leading alignment, the labels are sized to fit their content:

![screen shot 2018-11-30 at 10 56 59 am](https://user-images.githubusercontent.com/2257493/49320549-11d27d80-f4b7-11e8-9f25-fb70229ed913.png)